bob-apps/bin/bobc.rs

```diff
@@ -78,54 +383,102 @@ async fn get(key: Vec<u8>, addr: Uri) {
         Ok(res) => {
             let res: tonic::Response<_> = res;
             let data: &Blob = res.get_ref();
-            info!("OK");
             info!("response meta: {:?})", res.metadata());
             info!("data len: {}, meta: {:?}", data.data.len(), data.meta);
+            let file = File::create(filename).await;
+            match file {
+                Ok(mut file) => match file.write_all(&data.data).await {
+                    Ok(()) => {}
+                    Err(e) => {
+                        error!("{:?}", e);
+                    }
+                },
+                Err(err) => {
+                    error!("{:?}", err);
+                }
+            }
         }
         Err(res) => {
-            info!("ERR");
+            error!("{:?}", res);
+        }
+    }
+}
+
+async fn exist(keys: Vec<Vec<u8>>, addr: Uri) {
+    let mut client = BobApiClient::connect(addr).await.unwrap();
+
+    let keys = keys.into_iter().map(|key| BlobKey { key }).collect();
+    let message = ExistRequest {
+        keys,
+        options: None,
+    };
+    let request = Request::new(message);
+    let res = client.exist(request).await;
+    match res {
+        Ok(res) => {
             info!("{:?}", res);
         }
+        Err(e) => {
+            error!("{:?}", e);
+        }
     }
 }
 
 fn get_matches<'a>() -> ArgMatches<'a> {
-    let key_arg = Arg::with_name("key").takes_value(true).required(true);
-    let uri_arg = Arg::with_name("uri")
+    let key_arg = Arg::with_name(KEY_ARG)
+        .short("k")
+        .long("key")
+        .value_name("KEY")
+        .takes_value(true);
+    let key_size_arg = Arg::with_name(KEY_SIZE_ARG)
+        .help("Size of the binary key")
         .takes_value(true)
-        .default_value("http://localhost:20000");
-    let size_arg = Arg::with_name("size")
+        .long("size")
+        .short("s")
```

Comment:
Bad name for the parameter. Long name should be more descriptive: `key-size`. Short name for this parameter is better to remove, because it is rarely used in practice

Reply:
Done: [bf6dda4](https://github.com/qoollo/bob/pull/562/commits/bf6dda4bcf011e209eaf8c25d7b5c9cde4657782)